packages/thunderstore-api/src/fetch/teamDetails.ts

```diff
@@ -0,0 +1,11 @@
+import { RequestConfig } from "../index";
+import { apiFetch } from "../apiFetch";
+
+export async function fetchTeamDetails(
+  config: RequestConfig,
+  teamName: string
+) {
+  const path = `api/cyberstorm/team/${teamName}/`;
```

Comment:
Would it improve caching if we'd use `${teamName.toLowerCase()}` here and in other similar places? I'm not going to implement it here though, it should be done in one place when we get around implementing the "route definitions" helper.

Reply:
Assuming we do caching on the load balancing layer for this view, yes it would